components/collapsible/_collapsible.scss

```diff
@@ -73,12 +75,54 @@
       outline-offset: 3px;
       outline: 3px solid $color-focus-outline;
       color: $color-links-hover;
-    } 
+    }
+  }
+}
+
+.collapsible--contained {
+  .collapsible__title {
+    border: 1px solid $color-borders;
+    margin-top: 0.5rem;
+    padding-right: 0.65rem;
+    padding-left: 2.5rem;
+    &:hover {
+      background-color: $color-block-hover;
```

Comment:
In reviewing it I think that we can do away with the `:hover` background colour for the header but retain the button hover state.